app/Services/Coordinates.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
I normally think of something under `App\Services` as classes that connect to external services, not POPOs. Maybe we just throw this under `App`? Or if you like it under `App\Services` because it's attached to `Geocoder`, maybe it goes under `App\Services\Geocoder`?

Reply:
I completely agree!  I'm not really sure why I put it in services.  Since it is acting as a value object I just moved it to `App\Casts`.